.github/workflows/ci.yml

```diff
@@ -10,21 +10,9 @@ jobs:
   tests:
     strategy:
       fail-fast: false
-      matrix:
-        os: [ubuntu-20.04, ubuntu-22.04]
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
-        exclude:
-          - os: ubuntu-22.04
-            python-version: "3.8"
-          - os: ubuntu-22.04
-            python-version: "3.9"
-          - os: ubuntu-20.04
-            python-version: "3.10"
-          - os: ubuntu-20.04
-            python-version: "3.11"
-    runs-on: ${{ matrix.os }}
-
-    name: Python ${{ matrix.python-version }} Tests
+    runs-on: ubuntu-22.04
```

Comment:
As Louise commented, ubuntu 24.04 is available so lets try moving forward to that whilst we are updating this.